kivymd/font_definitions.py

```diff
@@ -12,13 +12,6 @@
 from kivymd import fonts_path
 
 fonts = [
-    {
-        "name": "Roboto",
-        "fn_regular": fonts_path + "Roboto-Regular.ttf",
-        "fn_bold": fonts_path + "Roboto-Bold.ttf",
-        "fn_italic": fonts_path + "Roboto-Italic.ttf",
-        "fn_bolditalic": fonts_path + "Roboto-BoldItalic.ttf",
-    },
```

Comment:
Actually, these fonts are used.

Reply:
these fonts are already exists in https://github.com/kivy/kivy/tree/master/kivy/data/fonts so we don't want to use separately.

Reply:
Have you compared these fonts? They differ.

Reply:
No, How do i compare them?

Reply:
Mostly it would be same.

Reply:
refs
https://github.com/kivy/kivy/blob/5e1311913722387242e1a3a48267333843a0e4a9/kivy/config.py#L867
https://github.com/kivy/kivy/blob/5e1311913722387242e1a3a48267333843a0e4a9/kivy/core/text/__init__.py#L94